packages/cli/src/services/experiment/ExperimentApiService.ts

```diff
@@ -13,15 +15,32 @@ export class ExperimentApiService {
     };
   }
 
-  async loadExperimentsList(deployment: string | undefined, limit = 1000): Promise<ExperimentModel[]> {
+  async loadFlagsList(deployment: string | undefined, limit = 1000): Promise<ExperimentFlagModel[]> {
     try {
       const response = await axios.get(`${EXPERIMENT_MANAGEMENT_API_BASE_URL}/experiments/list?limit=${limit}`, {
         headers: this.headers,
       });
 
-      const experiments: ExperimentModel[] = response.data.experiments; // eslint-disable-line prefer-destructuring
+      const experiments: ExperimentFlagModel[] = response.data.experiments; // eslint-disable-line prefer-destructuring
 
-      return deployment ? experiments.filter(e => e.key.startsWith(deployment)) : experiments;
+      // return deployment ? experiments.filter(e => e.key.startsWith(deployment)) : experiments;
+      return deployment ? experiments.filter(e => e.deployments.includes(deployment)) : experiments;
+    }
+    catch (e) {
+      return [];
+    }
+  }
+
+  async loadFlagsList2(deployment: string | undefined, limit = 1000): Promise<ExperimentConfigModel[]> {
```

Comment:
Minor, maybe a better name like `loadConfigsList` to make it easier to understand the diff between two methods.

Reply:
Haha ya, I am deleting this. I was waiting to merge in my other open PR since their was some changes in similar areas and I wanted to avoid conflicts. I will clean this up.